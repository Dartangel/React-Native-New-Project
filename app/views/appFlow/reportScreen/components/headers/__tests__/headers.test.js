import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Headers from '../headers';

describe('Headers', () => {
    test('Headers snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<Headers />);
        expect(snapshot).toMatchSnapshot();
    });
});