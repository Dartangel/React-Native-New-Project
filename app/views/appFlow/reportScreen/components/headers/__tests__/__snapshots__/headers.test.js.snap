// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headers Headers snapshot 1`] = `
<View
  style={
    Object {
      "flex": 0.5,
      "justifyContent": "center",
      "paddingLeft": 15,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#ffffff",
        "fontSize": 20,
      }
    }
  />
</View>
`;
